.gitignore

```diff
@@ -128,4 +128,5 @@ dist
 
 # Ignore Intellij project-specific directory
 /.idea/
-*.iml
\ No newline at end of file
+*.iml
+yarn.lock
```

Comment:
`yarn.lock` skal ikke legges i gitignore, og for i vårt tilfelle vil Ci-pipelinen faktisk feil, (ettersom vi bruker `--frozen-lockfile`